gangplank/go.mod

```diff
@@ -3,13 +3,15 @@ module github.com/coreos/gangplank
 go 1.14
 
 require (
+	github.com/containers/libpod v1.9.3
+	github.com/containers/storage v1.18.2
```

Comment:
Eek.  Is that really required just to speak the podman API?  That's a huge surface to vendor in, related to the new build deps.

Reply:
Yup. And _far_ worse is using the CLI. On the plus side, this kills any idea of using static binaries, so there's that.

Reply:
Doesn't podman implement the docker API?  And I'd expect Docker has a "just the API protocol" module?